content/02.body.md

```diff
@@ -156,6 +156,28 @@ The full source code for our method of degree-preserving network permutation has
 
 ## Results
 
+### Node degree bias
+
+We found examples of node degree bias in the PPI and TF-TG networks we investigated.
+Figure {@fig:degree-bias} shows node degree in separate networks for the same type of data.
+For the PPI networks, the literature-derived network has a larger mean degree and a longer tail than the systematic network, while in the TF-TG networks this relationship is reversed.
+Because the TF-TG network contained far more transcription factors than target genes (144 and 1406, respectively), the distributions of target degrees were far more compact than those of source degrees.
+Unlike the PPI and TF-TG networks, the co-authorship networks, which were split by date of first co-authorship, did not exhibit a great difference in their degree distributions.
+All three types of networks (PPI, TF-TG, and co-authorship) exhibit degree imbalance to varying extents.
+These results indicate that, depending on the methods by which the represented data were generated, networks of the same type of data may have overall degree distributions that differ greatly (Figure {@fig:degree-bias}A), and they may even assign very different degree to the same nodes (Figure {@fig:degree-bias}B).
+
+A prediction task that uses only one type of network is challenging because degree distributions can vary so greatly within a single domain.
```

Comment:
```suggestion
A prediction task that uses only one type of network is challenging because degree distributions vary greatly within a single domain.
```